internal/services/appservice/helpers/function_app_slot_schema.go

```diff
@@ -0,0 +1,1267 @@
+						string(web.ManagedPipelineModeIntegrated),
+					}, false),
+					Description: "The Managed Pipeline mode. Possible values include: `Integrated`, `Classic`. Defaults to `Integrated`.",
+				},
+
+				"pre_warmed_instance_count": {
+					Type:        pluginsdk.TypeInt,
+					Optional:    true,
+					Computed:    true, // Variable defaults depending on plan etc
+					Description: "The number of pre-warmed instances for this function app. Only affects apps on an Elastic Premium plan.",
+				},
+
+				"remote_debugging_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Should Remote Debugging be enabled. Defaults to `false`.",
+				},
+
+				"remote_debugging_version": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Computed: true,
+					ValidateFunc: validation.StringInSlice([]string{
+						"VS2017",
+						"VS2019",
+					}, false),
+					Description: "The Remote Debugging Version. Possible values include `VS2017` and `VS2019`",
+				},
+
+				"runtime_scale_monitoring_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Description: "Should Functions Runtime Scale Monitoring be enabled.",
+				},
+
+				"scm_type": {
+					Type:        pluginsdk.TypeString,
+					Computed:    true,
+					Description: "The SCM Type in use by the Windows Function App.",
+				},
+
+				"use_32_bit_worker": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     true,
+					Description: "Should the Windows Web App use a 32-bit worker.",
+				},
+
+				"websockets_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Should Web Sockets be enabled. Defaults to `false`.",
+				},
+
+				"ftps_state": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  string(web.FtpsStateDisabled),
+					ValidateFunc: validation.StringInSlice([]string{
+						string(web.FtpsStateAllAllowed),
+						string(web.FtpsStateDisabled),
+						string(web.FtpsStateFtpsOnly),
+					}, false),
+					Description: "State of FTP / FTPS service for this function app. Possible values include: `AllAllowed`, `FtpsOnly` and `Disabled`. Defaults to `Disabled`.",
+				},
+
+				"health_check_path": {
+					Type:        pluginsdk.TypeString,
+					Optional:    true,
+					Description: "The path to be checked for this function app health.",
+				},
+
+				"health_check_eviction_time_in_min": { // NOTE: Will evict the only node in single node configurations.
+					Type:         pluginsdk.TypeInt,
+					Optional:     true,
+					Computed:     true,
+					ValidateFunc: validation.IntBetween(2, 10),
+					Description:  "The amount of time in minutes that a node is unhealthy before being removed from the load balancer. Possible values are between `2` and `10`. Defaults to `10`. Only valid in conjunction with `health_check_path`",
```

Comment:
these are not tied together,`health_check_path` provides an optional "thing" to check for health, otherwise the check just makes sure it gets a `200` from the deployed site.